darshan_logs/hdf5_diagonal_write_only/README.md

```diff
@@ -0,0 +1,22 @@
+Set of example darshan logs that contain `H5D` and/or `H5F` module data
+to allow for "ground truth" verification for various data aggregation
+functions in the new Python summary report.
+
+The example application code used to generate these logs along with
+some of the execution details are available here:
+https://github.com/nawtrey/darshan_hdf5_testing
+
+A brief summary of each log's contents:
+
+| Log                                   | Modules                            | Description                                      |
+|---------------------------------------|------------------------------------|--------------------------------------------------|
+| `hdf5_file_opens_only`                | `POSIX`, `H5F`                     | 3 ranks open files                               |
+| `hdf5_diagonal_write_1_byte_dxt`      | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 1 byte each           |
+| `hdf5_diagonal_write_bytes_range_dxt` | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 10-100 bytes each     |
+| `hdf5_diagonal_write_half_flush_dxt`  | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 1 byte, 5 ranks flush |
+| `hdf5_diagonal_write_half_ranks_dxt`  | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, 5 ranks write 1 byte        |
+
```

Comment:
Oh, another somewhat unusual thing from a conventional HPC system log perspective, is that it looks like HDF5 as configured here is not using MPI-IO?  That's probably a notable thing to point out too as a characteristic of these logs.  It was a little surprising to me to see POSIX and HDF but not MPIIO in the modules.

Reply:
That's a good point. I'm not too familiar with HDF5 configuration so it didn't occur to me that it was strange. I can make a note of that though, if it's uncommon maybe it's good to have an example.